src/app.ts

```diff
@@ -99,6 +99,45 @@ async function app() {
 
   logger.info(`Starting ${getVersion()}`);
 
+  logger.info('Arguments:');
+  if (programOptions.logLevel) {
+    logger.info(`  --log-level ${programOptions.logLevel}`);
+  }
+  if (programOptions.dependency) {
+    logger.info(`  --dependency ${programOptions.dependency}`);
+  }
+  if (programOptions.style) {
+    logger.info(`  --style ${programOptions.style}`);
+  }
+  if (programOptions.fshingTrip) {
+    logger.info('  --fshing-trip');
+  }
+  if (programOptions.installedSushi) {
+    logger.info('  --installed-sushi');
+  }
+  if (programOptions.indent) {
+    logger.info('  --indent');
+  }
+  if (programOptions.installedSushi) {
+    logger.info('  --installed-sushi');
+  }
+  if (programOptions.metaProfile) {
+    logger.info(`  --meta-profile ${programOptions.metaProfile}`);
+  }
+  if (programOptions.aliasFile) {
+    logger.info(`  --alias-file ${programOptions.aliasFile}`);
+  }
+  if (programOptions.alias) {
+    logger.info('  --no-alias');
+  }
+  if (programOptions.useFHIRVersion) {
+    logger.info(`  --useFHIRVersion ${programOptions.useFHIRVersion}`);
+  }
+  if (programOptions.out) {
+    logger.info(`  --out ${path.resolve(programOptions.out)}`);
+  }
```

Comment:
This is just a question for now, but I realized that the `ensureOutputDir` function always logs the output director. Therefore, when you use the `-o --out` option, we end up logging the output directory twice - once here and once from within the `ensureOutputDir`. SUSHI's `ensureOutputDir` function works a little differently, so it only logs the output directory if it isn't set using `-o --out`, so it is always logged and only logged once. To make that work in GoFSH, we would need to update how `ensureOutputDir` works. Is that something we want to do? It might be a little out of scope of this PR. Or do we not mind the duplicate log? One does log the full path and one logs a relative path, so it might be okay to keep both. I'd like other opinions before actually asking for a change.

Reply:
I changed the --out log message to not include the whole file directory, do you think this approach is a bit better? 

`if (programOptions.out) {
    logger.info('  --out');
  }`

Reply:
Hm I think it would be good to keep the file path. Especially since it's the full resolved path, I think it's good context to display. It also aligns with SUSHI.

After thinking about it more, I think it's better to err on the side of more information. So I think rather than update how `ensureOutputDir` works, we can just log the path twice. So if you wouldn't mind, can you add back in the `path.resolve` portion?

Reply:
Sure - I'll keep it in.

Reply:
Thanks for swapping it back!